docs/source/guidelines.rst

```diff
@@ -1,48 +1,200 @@
 .. _Guidelines:
 
-Guidelines for Developers
-*************************
+Guidelines for Gammasim-tools Developers
+****************************************
 
-This section is meant for developers.
+This section provides help and guidelines for developers of Gammasim-tools.
+If you want to contribute to Gammasim-tools, please use one of the contact points listed at the entry page of this documentation.
+In general, please take note of the `ctapipe Development Guidelines <https://cta-observatory.github.io/ctapipe/development/index.html>`_.
```

Comment:
If you don't mind I will add explicitly a mention here that these guidelines are from ctapipe.